daal4py/sklearn/neighbors/_classification.py

```diff
@@ -58,11 +59,15 @@ def daal4py_classifier_predict(estimator, X, base_predict):
     except ValueError:
         fptype = None
 
-    if daal_model is not None and fptype is not None and not sp.issparse(X):
-        logging.info(
-            "sklearn.neighbors.KNeighborsClassifier"
-            ".predict: " + get_patch_message("daal"))
+    _patching_status = PatchingConditionsChain(
+        "sklearn.neighbors.KNeighborsClassifier.predict")
+    _dal_ready = _patching_status.and_conditions([
+        (daal_model is not None, "onedal model was not trained"),
+        (fptype is not None, "unknown fptype"),
```

Comment:
3-rd variant of floating point type message formulation, please align in all places